Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -855,7 +856,15 @@ getFleetVehicleAssociation _merchantShortId fleetOwnerId mbLimit mbOffset = do
 
         (driverName, driverId) <- case currentActiveDriver of
           Just driver -> pure (driver.firstName <> " " <> fromMaybe "" driver.lastName, Just driver.id.getId)
-          Nothing -> pure ("", Nothing)
+          Nothing -> do
+            latestAssociation <- QRCAssociation.findLatestLinkedByRCId vrc.id now
+            case latestAssociation of
+              Just latestAssociation' -> do
+                driver <- QPerson.findById latestAssociation'.driverId
+                case driver of
```

Comment:
Not good way of writing code when you have to handle Maybe with empty string. Handle it better way.

Reply:
sure i will fix it.